recipes/rapidyaml/all/conanfile.py

```diff
@@ -25,73 +29,78 @@ class RapidYAMLConan(ConanFile):
         "with_default_callbacks": True,
         "with_tab_tokens": False,
     }
-    generators = "cmake", "cmake_find_package_multi"
 
-    _compiler_required_cpp11 = {
-        "Visual Studio": "13",
-        "gcc": "6",
-        "clang": "4",
-    }
+    @property
+    def _minimum_cpp_standard(self):
+        return 11
 
     @property
-    def _source_subfolder(self):
-        return "source_subfolder"
+    def _compilers_minimum_version(self):
+        return {
+            "Visual Studio": "13",
```

Comment:
I think you can remove this line.

Reply:
@jwillikers 
Thank you for recommendation.
I will try to remove these lines.